node/narwhal/tests/common/trans.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the snarkOS library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use async_trait::async_trait;
+use indexmap::IndexMap;
+use snarkos_node_narwhal_ledger_service::{CoreLedgerService, LedgerService};
+use snarkvm::{
+    ledger::{
+        block::{Block, Transaction},
+        coinbase::{ProverSolution, PuzzleCommitment},
+        committee::Committee,
+        narwhal::{Data, Subdag, Transmission, TransmissionID},
+        store::ConsensusStorage,
+        Ledger,
+    },
+    prelude::{narwhal::BatchCertificate, Field, Network, Result},
+};
+use std::{fmt, ops::Range};
+
+pub struct TranslucentLedgerService<N: Network, C: ConsensusStorage<N>> {
```

Comment:
nit: can you move this under `ledger-service` with a `cfg(feature = "test")` guard and `test = [ ]` feature flag?

Reply:
moved